core/domain/exp_services.py

```
@@ -3189,22 +3095,15 @@ def get_scaled_average_rating(ratings: Dict[str, int]) -> float:
     """
-    exploration_summaries = exp_fetchers.get_exploration_summaries_matching_ids(
-        exp_ids
-    )
-    search_services.index_exploration_summaries(
-        [
-            exploration_summary
-            for exploration_summary in exploration_summaries
-            if exploration_summary is not None
-        ]
-    )
+    pass
```

Ruff (0.16.3)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)